README.md

```diff
@@ -1,2 +1,24 @@
-# Functional.CQS
-Defines CQS-style contracts and provides AOP facilities over those contracts using [Functional](https://github.com/JohannesMoersch/Functional) primitives
+# Functional.CQS.AOP
```

Comment:
This repo is called `Functional.CQS`, not `Functional.CQS.AOP` Also, it looks like the repo contains both projects.

Does it make sense to cover both projects/NuGet packages in this README? For example, this file could start with:
```
# Functional.CQS
Defines [CQS](docs/cqs_primer.md)-style contracts using [Functional](https://github.com/JohannesMoersch/Functional) primitives.

# Functional.CQS.AOP

`Functional.CQS.AOP` is a set of NuGet packages that ....

```

Reply:
For sure.  The documentation is just a copy of something I already wrote plus a string replace, so I will make the formatting change as you've suggested.

Reply:
Done